app/src/main/res/layout/play_set_of_verses.xml

```diff
@@ -33,4 +33,11 @@
     app:np_fadingEdgeEnabled="true"
     app:np_wrapSelectorWheel="true"
     />
+
+  <CheckBox
+      android:id="@+id/checkBoxInfiniteRange"
```

Comment:
Looks like you are using 4-space tabbing, Quran app uses 2-space tabbing.

Reply:
Although i was using 4-space tabbing, this is probably the [CONTINUATION_INDENT_SIZE](https://github.com/quran/quran_android/blob/eb557e823bd2eb48f064f19108d906aa787d4955/quran_android-code_style.xml#L350), and reformatting the file increases all other lines above like this one :)